CTP/common/ext/run_jdbc.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# 
+# Copyright (c) 2016, Search Solution Corporation. All rights reserved.
+# 
+# Redistribution and use in source and binary forms, with or without 
+# modification, are permitted provided that the following conditions are met:
+# 
+#   * Redistributions of source code must retain the above copyright notice, 
+#     this list of conditions and the following disclaimer.
+# 
+#   * Redistributions in binary form must reproduce the above copyright 
+#     notice, this list of conditions and the following disclaimer in 
+#     the documentation and/or other materials provided with the distribution.
+# 
+#   * Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products 
+#     derived from this software without specific prior written permission.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+# INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+#
+
+ctp_jdbc_test_conf=""
+
+function run_jdbc {
+    
+    if [ -f ${CTP_HOME}/conf/jdbc_template.conf ];then
+	ctp_jdbc_test_conf=${CTP_HOME}/conf/jdbc_template.conf
+    else
+	ctp_jdbc_test_conf=${CTP_HOME}/conf/jdbc.conf
+    fi
+
+    if [ "$BUILD_TYPE" != 'general' -a "$BUILD_TYPE" != 'debug' ]; then
+	echo ""
+        echo "The current CTP does not support $BUILD_TYPE testing!"
+	exit 
+    elif [ "$BUILD_SCENARIOS" != "jdbc" -a "$BUILD_SCENARIOS" == "jdbc_debug" ]; then
```

Comment:
as is:
elif [ "$BUILD_SCENARIOS" != "jdbc" -a "$BUILD_SCENARIOS" == "jdbc_debug" ]; then

to be:
1. elif -> if
2. 'jdbc_debug' is acceptable one.